aip-console-tools-core/src/main/java/com/castsoftware/aip/console/tools/core/utils/VersionInformation.java

```diff
@@ -0,0 +1,511 @@
+package com.castsoftware.aip.console.tools.core.utils;
+
+import com.castsoftware.aip.console.tools.core.exceptions.InvalidVersionException;
+import jdk.nashorn.internal.runtime.regexp.joni.exception.InternalException;
```

Comment:
?????

Reply:
Was automatically added but not used. I'll remove it